src/rompy/backends/config.py

```diff
@@ -284,5 +284,118 @@ def model_post_init(self, __context) -> None:
     )
 
 
+class SlurmConfig(BaseBackendConfig):
+    """Configuration for SLURM cluster execution."""
+
+    model_type: Literal["slurm"] = Field(
+        "slurm", 
+        description="The backend type."
+    )
+    queue: str = Field(
```

Comment:
This probably should be optional. Typically in our slurm configuration I do not specify the queue and instead slurm automatically uses the preconfigured default queue

Reply:
Noted, I'll change that